src/config.c

```diff
@@ -2601,6 +2601,71 @@ int allowProtectedAction(int config, client *c) {
            (config == PROTECTED_ACTION_ALLOWED_LOCAL && islocalClient(c));
 }
 
+
+static int setConfigLatencyTrackingInfoPercentilesOutputOption(typeData data, sds *argv, int argc, const char **err) {
+    UNUSED(data);
+    server.latency_tracking_info_percentiles = NULL;
+    server.latency_tracking_info_percentiles_len = argc;
+
+    /* Special case: treat single arg "" as zero args indicating empty percentile configuration */
+    if (argc == 1 && !strcasecmp(argv[0],""))
```

Comment:
case sensitive comparison is really excessive for this.
```suggestion
    if (argc == 1 && sdslen(argv[0]) == 0)
```